### ISSUE TITLE
Wrong conda panda-dev env filepath in "Creating a Python Environment" docs

### ISSUE THREAD

**Issue by enricorotundo** [pandas-dev/pandas#24384]
One of the first steps in the Contributing to Pandas guide is [Creating a Python Environment](http://pandas.pydata.org/pandas-docs/stable/contributing.html#creating-a-python-environment). The first line regers to a yaml file inside `ci/` but that files seems to have been [moved to the project root](https://github.com/pandas-dev/pandas/blob/master/environment.yml).
```
# Create and activate the build environment
conda env create -f ci/environment-dev.yaml
...
```
Should be
```
# Create and activate the build environment
conda env create -f environment.yml
...
```

**Comment by WillAyd** [comment]
Yea I can see where this is confusing because those docs don't point to the DEV version. I think this was resolved in another PR already though

**Comment by WillAyd** [comment]
@jamesmyatt

**Comment by TomAugspurger** [comment]
I think this is unavoidable right now. We likely won't be moving it again in the near future.

**Comment by TomAugspurger** [comment]
This will be resolved when we do the 0.24 release (coming soon).

**Comment by enricorotundo** [comment]
My bad sorry, I was sure I had checked the dev docs before opening this.
Btw, #23883 is related.
